README.md

```diff
@@ -13,66 +13,127 @@ Auto completion plugin for nvim.
 # Usage
 
 ```viml
-let g:compe_enabled = v:true
-let g:compe_debug = v:false
-let g:compe_min_length = 1
-let g:compe_auto_preselect = v:false
-let g:compe_throttle_time = 120
-let g:compe_source_timeout = 200
-let g:compe_incomplete_delay = 400
+if s:viml
+  let g:compe = {}
+  let g:compe.enabled = v:true
+  let g:compe.debug = v:false
+  let g:compe.min_length = 1
+  let g:compe.auto_preselect = v:true
+  let g:compe.throttle_time = 120
+  let g:compe.source_timeout = 200
+  let g:compe.incomplete_delay = 400
+
+  let g:compe.source = {}
```

Comment:
`sources` is better here instead of `source`

Reply:
Yes. I wondered about source vs sources.

It's just curiosity.
In coc.nvim, it can be configure like as below.

It is not `languageservers`. What do you feel it?

```
  call coc#config('languageserver', {
  \   'sumneko_lua': {
  \     'command': expand('~/Develop/Repos/lua-language-server/bin/macOS/lua-language-server'),
  \     'args': ['-E', expand('~/Develop/Repos/lua-language-server/main.lua')],
  \     'filetypes': ['lua'],
  \     'settings': {
  \       'Lua': {
  \         'runtime': {
  \           'version': 'LuaJIT',
  \         },
  \         'completion': {
  \           'callSnippet': 'Replace',
  \         },
  \         'diagnostics': {
  \           'enable': v:true,
  \           'globals': [
  \             'vim', 'describe', 'it', 'before_each', 'after_each'
  \           ],
  \         },
  \         'workspace': {
  \           'library': s:lua_library,
  \         },
  \       },
  \     }
  \   }
  \ })
```

Reply:
I just seen that, srry. 

when we are referring to an object (like the case above) then its good to use single word, however when we refer to a list of stuff is good to have plural because it means that the content is a list of the same type.